### PR TITLE
ADR-942 dynamic section heading text for adjustment journey

### PR DESCRIPTION
**Pull request by JemmaEdwards**

### CODE REVIEW

File: app/controllers/adjustment/AdjustmentVolumeController.scala [hunk 1]
```diff
@@ -47,25 +50,37 @@ class AdjustmentVolumeController @Inject() (
   val form = formProvider()
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val preparedForm = request.userAnswers.get(AdjustmentVolumePage) match {
-      case None        => form
-      case Some(value) => form.fill(value)
+    request.userAnswers.get(CurrentAdjustmentEntryPage) match {
+      case None                                  => Redirect(controllers.routes.JourneyRecoveryController.onPageLoad())
```
Comment (sini-george) [inline app/controllers/adjustment/AdjustmentVolumeController.scala:54]: Could you please add a test for the None case

File: app/controllers/adjustment/AlcoholByVolumeController.scala [hunk 2]
```diff
@@ -49,20 +50,27 @@ class AlcoholByVolumeController @Inject() (
   val form = formProvider()
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val abv          = request.userAnswers.get(CurrentAdjustmentEntryPage).flatMap(_.abv)
-    val preparedForm = abv match {
-      case None                  => form
-      case Some(alcoholByVolume) => form.fill(alcoholByVolume.value)
+    request.userAnswers.get(CurrentAdjustmentEntryPage) match {
+      case None                               => Redirect(controllers.routes.JourneyRecoveryController.onPageLoad())
```
Comment (sini-george) [inline app/controllers/adjustment/AlcoholByVolumeController.scala:54]: Could you please add a test for the None case

File: test/controllers/adjustment/AdjustmentVolumeControllerSpec.scala [hunk 3]
```diff
@@ -23,16 +23,19 @@ import navigation.{AdjustmentNavigator, FakeAdjustmentNavigator}
 import org.mockito.ArgumentMatchers.any
 import org.mockito.Mockito.when
 import org.scalatestplus.mockito.MockitoSugar
-import pages.adjustment.AdjustmentVolumePage
+import pages.adjustment.{AdjustmentVolumePage, CurrentAdjustmentEntryPage}
```
Comment (sini-george) [inline test/controllers/adjustment/AdjustmentVolumeControllerSpec.scala:26]: Could you please remove the unused imports like AdjustmentVolumePage and Try

File: app/controllers/adjustment/AdjustmentVolumeController.scala (hunk 1 above)
Comment (JemmaEdwards) [reply app/controllers/adjustment/AdjustmentVolumeController.scala:54]: test changed

File: app/controllers/adjustment/AlcoholByVolumeController.scala (hunk 2 above)
Comment (JemmaEdwards) [reply app/controllers/adjustment/AlcoholByVolumeController.scala:54]: test changed

File: test/controllers/adjustment/AdjustmentVolumeControllerSpec.scala (hunk 3 above)
Comment (JemmaEdwards) [reply test/controllers/adjustment/AdjustmentVolumeControllerSpec.scala]: removed!

**Review by sini-george — APPROVED** [review]
